packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -189,7 +195,7 @@ export function createAddAzureResourceQuestion(
   alreadhHaveSQL: boolean,
   alreadyHaveAPIM: boolean
 ): MultiSelectQuestion {
-  const options: OptionItem[] = [AzureResourceFunction];
+  const options: OptionItem[] = [AzureResourceFunction, AzureResourceKeyVault];
```

Comment:
should we check whether user already added key vault?

Reply:
Good catch!